installing/installing_aws/upi/upi-aws-preparing-to-install.adoc

```diff
@@ -0,0 +1,46 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upi-aws-preparing-to-install"]
+= Preparing to install a cluster on AWS
+include::_attributes/common-attributes.adoc[]
+:context: upi-aws-preparing-to-install
+
+toc::[]
+
+You prepare to install an {product-title} cluster on AWS by completing the following steps:
+
+* Verifying internet connectivity for your cluster.
+
+* xref:../../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configuring an AWS account].
+
+* Downloading the installation program.
++
+[NOTE]
+====
+If you are installing in a disconnected environment, you extract the installation program from the mirrored content. For more information, see xref:../../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installing-mirroring-installation-images[Mirroring images for a disconnected installation].
+====
+* Installing the {oc-first}.
++
+[NOTE]
+====
+If you are installing in a disconnected environment, install `oc` to the mirror host.
+====
+* Generating an SSH key pair. You can use this key pair to authenticate into the {product-title} cluster's nodes after it is deployed.
+
+* xref:../../../installing/installing_aws/upi/upi-aws-installation-reqs#upi-aws-installation-reqs[Preparing the user-provisioned infrastructure.]
+
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, xref:../../../installing/installing_aws/ipi/installing-aws-customizations.adoc#manually-create-iam_installing-aws-customizations[manually creating long-term credentials for AWS] or xref:../../../installing/installing_aws/ipi/installing-aws-customizations.adoc#installing-aws-with-short-term-creds_installing-aws-customizations[configuring an AWS cluster to use short-term credentials] with Amazon Web Services Security Token Service (AWS STS).
```

Comment:
Yeah, your understanding is correct. It's attempting to say, if you can't access the IAM API, or you have security concerns with admin credentials, you can create them yourself or use short term credentials instead. 

This will probably be reworked in a future PR :) Thanks